### PR TITLE
csclient/params: add CanWriteResponse

### PR DESCRIPTION
**Pull request by rogpeppe**

### CODE REVIEW

**Comment by rogpeppe** [comment]
:shipit:

File: csclient/params/params.go [hunk 1]
```diff
@@ -281,6 +281,12 @@ type CanIngestResponse struct {
 	CanIngest bool
 }
 
+// CanWriteResponse holds the result of an id/meta/can-write GET request.
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetacan-write
```
Comment (mhilton) [inline csclient/params/params.go]: This URL doesn't work, can we have one which does please.

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmrepo-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/charmrepo/322/

**Pull request merged**